support-frontend/assets/pages/paper-subscription-landing/components/content/deliveryTab.jsx

```diff
@@ -40,7 +40,7 @@ const ContentDeliveryFaqBlock = ({
   <Content
     border={paperHasDeliveryEnabled()}
     image={<GridImage
-      gridId="printCampaignHD"
+      gridId={useDigitalVoucher ? 'printCampaignHDiMovo' : 'printCampaignHD'}
```

Comment:
I think this is getting updated and isn't conditional on useDigitalVoucher (could be misunderstanding though!)